sdk/tables/Azure.Data.Tables/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Remove this file, it doesn't define anything except inclusion of higher level `Directory.Build.props` that happens automatically.